napari/_qt/_qapp_model/qactions/_debug.py

```diff
@@ -50,24 +50,19 @@ def _start_trace_dialog(qt_viewer: QtViewer) -> None:
 
 def _start_trace(path: str) -> None:
     """Start recording a trace file."""
-    # `DummyTimer` has no `start_trace_file`
-    start_trace = getattr(perf.timers, 'start_trace_file', None)
-    if start_trace:
+    if start_trace := perf.timers.start_trace_file:
         start_trace(path)
```

Comment:
why not just `perf.timers.start_trace_file()`?

Reply:
:facepalm: yes I am not thinking